packages/devtools_app/lib/src/screens/debugger/debugger_model.dart

```diff
@@ -420,9 +420,11 @@ Future<void> buildVariablesTree(
       start += count;
     }
   } else if (instanceRef != null && serviceManager.service != null) {
+    final variableId = variable.ref!.isolateRef!.id!;
     try {
-      final dynamic result =
-          await _getObjectWithRetry(instanceRef.id!, variable);
+      final dynamic result = await serviceManager.service!.getObject(
+          variableId, instanceRef.id!,
+          offset: variable.offset, count: variable.childCount);
```

Comment:
nit: add a trailing comma after `variable.childCount` --> `variable.childCount,`. This will allow the dart formatter to format the code properly. Generally, whenever a method call has to break to a new line, you should add a trailing comma. We have a lint check for this enabled for the DevTools codebase, so when a trailing comma is needed, you should see a warning in the Dart Analysis (IntelliJ) or Problems (VS Code) section of your IDE.